File: examples/mq/subscribe/go/subscribeToATopic.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/mq"
)

// Subscribe to messages for a given topic.
func SubscribeToAtopic() {
	mqService := mq.NewMqService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := mqService.Subscribe(&mq.SubscribeRequest{
		Topic: "events",
	})
	fmt.Println(rsp, err)
}

File: examples/mq/subscribe/node/subscribeToATopic.js
const { MqService } = require("micro-js-client/mq");

// Subscribe to messages for a given topic.
async function subscribeToAtopic() {
  let mqService = new MqService(process.env.MICRO_API_TOKEN);
  let rsp = await mqService.subscribe({
    topic: "events",
  });
  console.log(rsp);
}

subscribeToAtopic();
